lib/linux_admin/dns.rb

```diff
@@ -0,0 +1,41 @@
+module LinuxAdmin
+  class Dns
+    attr_accessor :filename
+    attr_accessor :nameservers
+    attr_accessor :search_order
+
+    def initialize(filename = "/etc/resolv.conf")
+      @filename = filename
+      reload
+    end
+
+    def reload
+      parse_file(File.read(@filename).split("\n"))
```

Comment:
`parse_file` clears them to start.  Would it look better to clear them first here?